CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## HEAD (Unreleased)
+
+- Update `@pulumi/aws` and `@pulumi/azure` dependencies to 1.0.0
```

Comment:
Should this be `@pulumi/aws` and `@pulumi/pulumi`?

Reply:
Indeed.